### DISCUSSION TITLE
Preview release v6.9.9

### DISCUSSION THREAD

**Discussion by arkypita** [arkypita/LaserGRBL#2249] [General]
This board is to discuss version 6.9.9 released in preview for supporters of the project.

[![v6.9.9 preview](http://img.youtube.com/vi/P2wsZlC67os/0.jpg)](https://www.youtube.com/watch?v=P2wsZlC67os "LaserGRBL v6.9.9 what's new")

**Comment by osbrook** [comment]
Both Windows defender and Norton report this files as dangerous and will not run the install.   Norton then removes the file.  Unable to test the new version (699)

**Comment by arkypita** [comment]
https://lasergrbl.com/does-lasergrbl-contain-viruses/
